lib/lib-storage/src/types.ts

```diff
@@ -51,8 +54,15 @@ export interface Configuration {
 
   /**
    * Optional abort controller for controlling this upload's abort signal externally.
+   *
+   * @deprecated use `abortSignal` instead.
+   */
+  abortController?: IAbortController;
+
+  /**
+   * Optional abort signal for controlling this upload's abort signal externally.
    */
-  abortController?: AbortController;
+  abortSignal?: globalThis.AbortSignal;
```

Comment:
The code technically supports `IAbortSignal` but since those are kinda deprecated, and this is a new feature, there is no point in adding more legacy here.